src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -146,144 +149,183 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
+                        "features_networks_zones",
+                        !formik.values.features_networks_zones,
+                      );
+                    }}
+                    checked={formik.values.features_networks_zones}
+                    disabled={
+                      isDefaultProject ||
+                      (isNonEmpty && hadFeaturesNetworkZones)
+                    }
+                  />
+                )}
+                {hasStorageBuckets && (
+                  <CheckboxInput
+                    id="features_storage_buckets"
+                    name="features_storage_buckets"
+                    label="Storage buckets"
+                    onChange={() => {
+                      ensureEditMode(formik);
+                      void formik.setFieldValue(
+                        "features_storage_buckets",
+                        !formik.values.features_storage_buckets,
+                      );
+                    }}
+                    checked={formik.values.features_storage_buckets}
+                    disabled={isDefaultProject || isNonEmpty}
+                  />
+                )}
                 <CheckboxInput
-                  id="features_storage_buckets"
-                  name="features_storage_buckets"
-                  label="Storage buckets"
+                  id="features_storage_volumes"
+                  name="features_storage_volumes"
+                  label="Storage volumes"
                   onChange={() => {
                     ensureEditMode(formik);
                     void formik.setFieldValue(
-                      "features_storage_buckets",
-                      !formik.values.features_storage_buckets,
+                      "features_storage_volumes",
+                      !formik.values.features_storage_volumes,
                     );
                   }}
-                  checked={formik.values.features_storage_buckets}
+                  checked={formik.values.features_storage_volumes}
                   disabled={isDefaultProject || isNonEmpty}
                 />
-              )}
-              <CheckboxInput
-                id="features_storage_volumes"
-                name="features_storage_volumes"
-                label="Storage volumes"
-                onChange={() => {
-                  ensureEditMode(formik);
-                  void formik.setFieldValue(
-                    "features_storage_volumes",
-                    !formik.values.features_storage_volumes,
-                  );
-                }}
-                checked={formik.values.features_storage_volumes}
-                disabled={isDefaultProject || isNonEmpty}
-              />
-            </>
-          )}
+              </>
+            )}
+          </div>
+
           <hr />
+          <StoragePoolSelector
+            id="default_instance_storage_pool"
+            name="default_instance_storage_pool"
+            value={formik.values.default_instance_storage_pool as string}
```

Comment:
```suggestion
            value={formik.values.default_instance_storage_pool}
```
With the stricter type, this cast should not be necessary.